src/pocketmine/command/defaults/TimeCommand.php

```diff
@@ -117,6 +117,7 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
 					break;
 				default:
 					$value = $this->getInteger($sender, $args[1], 0);
+					break;
```

Comment:
Is this just for style consistency?

Reply:
Not just style. It's not likely someone will add a case after the default, but in case we do for some reason, this will save that someone a headache. A minute spent now can save an hour later.